data/filter.go

```diff
@@ -72,75 +72,76 @@ var (
 	// Bulletin - Search information specific for statistical bulletins
 	Bulletin = ContentType{
 		LocaliseKeyName: "StatisticalBulletin",
-		Type:            "bulletin",
-		SubTypes:        []string{"bulletin"},
+		Group:           "bulletin",
+		Types:           []string{"bulletin"},
 	}
 
 	// Article - Search information specific for articles
 	Article = ContentType{
 		LocaliseKeyName: "Article",
-		Type:            "article",
-		SubTypes:        []string{"article", "article_download"},
+		Group:           "article",                               //Type
```

Comment:
Probably need to remove comments on this line and the one below

Reply:
Done